api/clever/__init__.py

```diff
@@ -264,7 +264,7 @@ def remote_patron_lookup(self, token):
         identifier = data.get('id', None)
 
         user_type = data.get('type', None)
-        self.log.info('identifier', result.to_dict())
+        self.log.info('identifier', result)
```

Comment:
This errors out due to it being `None`, we should log `data` to see what we're really getting back from Clever.

Reply:
Or log `result`, since data might be `{}` when we set it.